README.adoc

```diff
@@ -41,19 +41,19 @@ There are five types of metric defined in the Dropwizard Metrics API:
 - A *timer* is basically a histogram of the duration of events coupled with a meter of the rate of the event occurrence. 
 
 ==== System and Java Virtual Machine metrics
-`system.cpu.load` (gauge):: The system load on the Jenkins master as reported by the JVM's Operating System JMX bean.
+`system.cpu.load` (gauge):: The system load on the Jenkins controller as reported by the JVM's Operating System JMX bean.
 _The calculation of system load is operating system dependent._ Typically this is the sum of the number of processes that are currently running plus the number that are waiting to run.
 This is typically comparable against the number of CPU cores.
 
-`vm.blocked.count` (gauge):: The number of threads in the Jenkins master JVM that are currently blocked waiting for a monitor lock.
+`vm.blocked.count` (gauge):: The number of threads in the Jenkins JVM that are currently blocked waiting for a monitor lock.
 
-`vm.count` (gauge):: The total number of threads in the Jenkins master JVM.
```

Comment:
so here this is just Jenkins JVM, in others you are saying on the `Jenkins Controller`

Reply:
Because there is only the one Jenkins JVM running on a controller it seemed redundant to call it the Jenkins controller JVM.
In other cases the text was referring to specifics about a controller, so I left it there.

Reply:
Perfect. Thank you @kerogers-cloudbees for the clarification.